scripts/system_tests_utils.ts

```diff
@@ -53,13 +53,22 @@ export const getEnvVar = (name: string): string | undefined =>
 	global.Cypress ? Cypress.env(name) : process.env[name];
 
 export const DEFAULT_SYSTEM_PROVIDER = 'http://localhost:8545';
+export const DEFAULT_SYSTEM_ENGINE = 'node';
 
 export const getSystemTestProvider = (): string =>
 	getEnvVar('WEB3_SYSTEM_TEST_PROVIDER') ?? DEFAULT_SYSTEM_PROVIDER;
 
-export const isHttp = getSystemTestProvider().startsWith('http');
-export const isWs = getSystemTestProvider().startsWith('ws');
-export const isIpc = getSystemTestProvider().includes('ipc');
+export const getSystemTestEngine = (): string =>
+	getEnvVar('WEB3_SYSTEM_TEST_ENGINE') ?? DEFAULT_SYSTEM_ENGINE;
+
+export const isHttp: boolean = getSystemTestProvider().startsWith('http');
+export const isWs: boolean = getSystemTestProvider().startsWith('ws');
+export const isIpc: boolean = getSystemTestProvider().includes('ipc');
+export const isChrome: boolean = getSystemTestEngine() === 'chrome';
```

Comment:
Would be nice to call browser and node as `Client` or `Environment` rather `Engine`.